corehq/apps/ota/views.py

```diff
@@ -377,11 +375,25 @@ def form_valid(self):
 
 @login_or_digest_or_basic_or_apikey()
 @require_GET
-def heartbeat(request, domain, id):
-    # mobile needs this. This needs to be revisited to actually work dynamically (Sravan June 7, 17)
-    for_app_id = request.GET.get('app_id', '')
-    return JsonResponse({
-        "app_id": for_app_id,
-        "latest_apk_version": {},
-        "latest_ccz_version": {}
-    })
+def heartbeat(request, domain, app_id):
+    """
+    An endpoint for CommCare mobile to get latest CommCare APK and app version
+        info. (Should serve from cache as it's going to be busy view)
+
+    'app_id' (that comes from URL) can be id of any version of the app
+    'app_id' (urlparam) is usually id of an app that is not a copy
```

Comment:
Yeah, it's not clear from spec I am also not sure whether it comes from `@unique_id`, but Aliza mentioned me that app_id urlparam is going to be the fixed app-id that doesn't change between versions, @amstone326 can you confirm where this urlparam app_id is pulled from when mobile makes this request?